data/sql/updates/pending_db_world/rev_1603401587698987500.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1603401587698987500');
+
+-- Ulduar Colossus bridge pathing fix --
+REPLACE INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`)  VALUES 
```

Comment:
please use `DELETE` and `INSERT` instead of `REPLACE`

https://www.azerothcore.org/wiki/Dealing-with-SQL-files

Reply:
fixed 🎃